daml-script/daml/Daml/Script.daml

```diff
@@ -731,3 +741,39 @@ submitTreeMulti actAs readAs cmds =
     commands = void cmds
     locations = getCallStack callStack
     continue = identity
+
+
+data User = User { id : Text, primaryParty : Party }
+
+data UserRight
+  = ParticipantAdmin
+  | CanActAs Party
+  | CanReadAs Party
+
+createUser : User -> [UserRight] -> Script User
+createUser = error ""
+
+getUser : Text -> Script User
```

Comment:
not quite sure if we want to expose user ids
1. As plain Text
2. As newtyped Text but constructor exposed
3. Opaquely with only getUser allowing a lookup.